src/components/application_manager/include/application_manager/message_helper.h

```diff
@@ -279,6 +284,16 @@ class MessageHelper {
   static smart_objects::SmartObjectList CreateAddCommandRequestToHMI(
       ApplicationConstSharedPtr app, ApplicationManager& app_mngr);
 
+  /**
+   * @brief Creates Message for sending to HMI for unsubscribe from specified
+   * VehicleData. Puts in vehicle data types into message according to keys.
+   * @param message_to_hmi - message for sending.
+   * @param vehicle_data - collection with vehicle data types for ubsubscribe.
+   */
+  static void CreateUbsubscriveVehicleDataMessageForHMI(
```

Comment:
@IGapchuk Is it correct name of method? Maybe you mean `CreateUnsubscribeVehicleDataMessageForHMI` ?

Reply:
Done. Method was renamed.